portal/src/components/Search/SearchBar.vue

```diff
@@ -0,0 +1,305 @@
+<template>
+  <section class="main__info_search">
+    <PreFilterSearch class="main__info_search__domain" @update:filter="onUpdateFilter" />
```

Comment:
I expected the SearchBar component to be named PreFilterSearch and this wrapper around the filter categories can still disappear in my opinion. It only wraps the FilterCategories, so why not place it here directly? Makes it more readable especially since PreFilterSearch is not actually doing anything with search, only something with filtering.